pikaday.js

```diff
@@ -407,11 +409,20 @@
             j = 1 + year + opts.yearRange;
         }
 
-        for (arr = []; i < j && i <= opts.maxYear; i++) {
-            if (i >= opts.minYear) {
-                arr.push('<option value="' + i + '"' + (i === year ? ' selected="selected"': '') + '>' + (i) + '</option>');
+        if (opts.yearOrderMinMax) {
+            for (arr = []; i < j && i <= opts.maxYear; i++) {
+                if (i >= opts.minYear) {
+                    arr.push('<option value="' + i + '"' + (i === year ? ' selected="selected"': '') + '>' + (i) + '</option>');
+                }
+            }
+        } else {
+            for (arr = []; j > i && j >= opts.minYear; j--) {
+                if (j <= opts.maxYear) {
+                    arr.push('<option value="' + j + '"' + (j === year ? ' selected="selected"': '') + '>' + (j) + '</option>');
```

Comment:
Can you find solution that don't repeat the `<option>` code? Now we would have to change it twice if we want to update the line.

Reply:
Please refer to [Pull Request 833](https://github.com/Pikaday/Pikaday/pull/833)